src/web/mjs/connectors/BentoManga.mjs

```diff
@@ -0,0 +1,79 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class bentomanga extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'bentomanga';
+        super.label = 'Bentô Manga';
+        this.tags = [ 'manga', 'webtoon', 'novel', 'french' ];
+        this.url = 'https://bentomanga.com';
+    }
+
+    async _getMangaFromURI(uri) {
+        let request = new Request(uri, this.requestOptions);
+        let data = await this.fetchDOM(request, 'h1'); 
```

Comment:
Way too lax CSS selector. You must make sure it will be unique. 

``` div.component-manga-title_main h1 ``` is a better candidate